docgen/assets/js/bindRunExamples.js

```diff
@@ -4,13 +4,14 @@ import capitalize from "lodash/capitalize";
 
 window.instantsearch = instantsearch;
 window.search = instantsearch({
-  indexName: "instant_search",
-  searchClient: algoliasearch("latency", "6be0576ff61c053d5f9a3225e2a90f76"),
-  urlSync: false,
+  indexName: window.searchConfig.indexName || "instant_search",
+  searchClient: algoliasearch(
+    window.searchConfig.appId || "latency",
+    window.searchConfig.apiKey || "6be0576ff61c053d5f9a3225e2a90f76"
+  ),
   searchParameters: {
-    hitsPerPage: 3
-  },
-  ...window.searchConfig
```

Comment:
It there a reason why we don't spread the rest of the arguments anymore?

Reply:
In the frontmatter, you can add `appId` and `apiKey` that get into `window.searchConfig`.

InstantSearch doesn't allow passing `appId` and `apiKey` directly, but require a search client.

We can still spread `window.searchConfig` without these two options.

Reply:
Note that we only take into account the options `appId`, `apiKey` and `indexName`: https://github.com/algolia/instantsearch.js/blob/feat/3.0-search-client/docgen/plugins/searchConfig.js#L14